### ISSUE TITLE
Feature request: encrypt data and use biometric unlock

### ISSUE THREAD

**Issue by Mrnofish** [SamAmco/track-and-graph#75]
Track & Graph can be used to collect sensitive information; it would be a good idea to let the user encrypt their data and provide biometric unlocking.

**Comment by SamAmco** [comment]
I agree with this however I don't have a lot of time to dedicate to this project right now so I'm not sure I will be able to get this done in the near future. I have found that I'm able to add bio-metric unlocking to the app via the android OS so you could try this? However the database is not encrypted.

**Comment by pipatron** [comment]
Should this not be strictly up to the operating system? I think it would get cluttered and inefficient if every application required its own encryption. A lot of apps can be used to collect sensitive information.

**Comment by SamAmco** [comment]
Like i say the database is not encrypted so this at least would need to be done app side. Most apps do add their own biometric/password unlock code also for better or worse.

**Comment by SamAmco** [comment]
Biometric unlock exists now. Data encryption does not, but can be a separate issue.
